risk/.htaccess

```diff
@@ -0,0 +1,41 @@
+Options +FollowSymLinks
+RewriteEngine on
+
+# Turn off MultiViews
+Options -MultiViews
+
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .json
+
+### Rewrite rules
+
+#RewriteRule ^$ http://ontology.tno.nl/saref [R=302,L]
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml 
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ https://hpruvost.pages.fraunhofer.de/semantic-models/risk/doc/index.html  [R=303,NE,L]
+
+# Rewrite rule to serve JSON-LD content from the vocabulary URI if requested
+#RewriteCond %{HTTP_ACCEPT} application/ld\+json
+#RewriteRule ^$ https://hpruvost.pages.fraunhofer.de/semantic-models/ontology.jsonld [R=303,L]
+
+# Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
+#RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
+#RewriteRule ^$ https://hpruvost.pages.fraunhofer.de/semantic-models/ontology.rdf [R=303,NE,L]
+
+# Rewrite rule to serve N-Triples content from the vocabulary URI if requested
+#RewriteCond %{HTTP_ACCEPT} application/n-triples
+#RewriteRule ^$ https://hpruvost.pages.fraunhofer.de/semantic-models/ontology.nt [R=303,L]
+
+# Rewrite rule to serve TTL content from the vocabulary URI if requested
+#RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.* 
+#RewriteRule ^$ https://hpruvost.pages.fraunhofer.de/semantic-models/ontology.ttl [R=303,NE,L]
+
+#default response: owl
+RewriteRule ^$ https://hpruvost.pages.fraunhofer.de/semantic-models/risk/doc/index.html [R=303,NE,L]
```

Comment:
Was this supposed to go to something other than HTML?  Similar for the `sense` rules.

Reply:
HTML is correct, the comment was misleading. 
I have corrected it.
Thx